test/protocol-buffer-conversion-test.ts

```diff
@@ -166,4 +182,49 @@ describe('Protocol Buffer Conversion', function(): void {
     // THEN the result is undefined
     assert.isUndefined(issuedCurrency)
   })
+
+  // CurrencyAmount tests
+  it('Convert CurrencyAmount with drops', function(): void {
+    // GIVEN a currency amount protocol buffer with an XRP amount.
+    const drops = '10'
+    const currencyAmountProto = new CurrencyAmount()
+    const xrpDropsAmountProto = new XRPDropsAmount()
+    xrpDropsAmountProto.setDrops(drops)
+    currencyAmountProto.setXrpAmount(xrpDropsAmountProto)
+
+    // WHEN the protocol buffer is converted to a native TypeScript type.
+    const currencyAmount = XRPCurrencyAmount.from(currencyAmountProto)
+
+    // THEN the result has drops set and no issued amount.
+    assert.isUndefined(currencyAmount?.issuedCurrency)
+    assert.equal(currencyAmount?.drops, drops)
+  })
+
+  it('Convert CurrencyAmount with Issued Currency', function(): void {
+    // GIVEN a currency amount protocol buffer with an issued currency amount.
+    const currencyAmountProto = new CurrencyAmount()
+    currencyAmountProto.setIssuedCurrencyAmount(testIssuedCurrency)
+
+    // WHEN the protocol buffer is converted to a native TypeScript type.
+    const currencyAmount = XRPCurrencyAmount.from(currencyAmountProto)
+
+    // THEN the result has an issued currency set and no amount.
```

Comment:
This 'THEN' could probably be phrased as "no xrp amount" or "no drops amount" which is slightly more specific.

Reply:
Apologies since I'm guessing you're copying me :)

Reply:
haha, I am! but I agree nonetheless